src/test/java/com/wikia/webdriver/common/core/helpers/ContentLoader.java

```diff
@@ -24,25 +22,15 @@ private ContentLoader() {}
   public static String loadWikiTextContent(String filename) {
     StringBuilder textContent = new StringBuilder();
     String separator = System.getProperty("line.separator");
-    Scanner scanner = null;
-
-    try {
-      scanner =
-          new Scanner(
-              new File(ClassLoader
-                  .getSystemResource(String.format(TEXT_FILES_PATH_FORMAT, filename)).getPath()),
-          "UTF-8");
-    } catch (FileNotFoundException e) {
-      PageObjectLogging.logError("File was not loaded", e);
-    }
-    try {
+    String path = String.format(TEXT_FILES_PATH_FORMAT, filename);
+    try (Scanner scanner = new Scanner(new File(ClassLoader.getSystemResource(path).getPath()),
+      "UTF-8")) {
       while (scanner.hasNextLine()) {
-        textContent.append(scanner.nextLine() + separator);
+        textContent.append(scanner.nextLine()).append(separator);
       }
-    } finally {
-      scanner.close();
+    } catch (NullPointerException | FileNotFoundException ex) {
+      throw new RuntimeException("Error when loading file!", ex);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00112)